presto-hive/src/main/java/com/facebook/presto/hive/HiveErrorCode.java

```diff
@@ -62,6 +62,7 @@
     HIVE_TABLE_DROPPED_DURING_QUERY(35, EXTERNAL),
     // HIVE_TOO_MANY_BUCKET_SORT_FILES(36) is deprecated
     HIVE_CORRUPTED_COLUMN_STATISTICS(37, EXTERNAL),
+    HIVE_NULL_COLUMN_NEGATIVE_STATISTICS(38, EXTERNAL),
```

Comment:
add Exception code HIVE_NULL_COLUMN_NEGATIVE_STATISTICS when we meet -1 for nullCount